pom.xml

```diff
@@ -40,6 +40,14 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```

Comment:
I see what you did here :laughing: 

Only thing is that you added the lombook dependency twice